nvm.sh

```diff
@@ -117,11 +117,17 @@ nvm_get_latest() {
 
 nvm_download() {
   local CURL_COMPRESSED_FLAG
+  local AUTH_HEADER_FLAG
+
+  if [ -n "$NVM_AUTH_HEADER" ]; then
+    AUTH_HEADER_FLAG="--header $NVM_AUTH_HEADER"
```

Comment:
```suggestion
  if [ -n "${NVM_AUTH_HEADER}" ]; then
    AUTH_HEADER_FLAG="--header \"${NVM_AUTH_HEADER}\" "
```
i think the added quotes will prevent people from being able to inject arbitrary curl/wget commands